go

```diff
@@ -59,7 +59,7 @@ def init
 end
 
 def update_gems
-  exec_cmd 'bundle update'
+  exec_cmd 'bundle install'
```

Comment:
Then perhaps you should add a `install_bundle` command and use that in the `build` and `serve` commands (or just call `bundle install` there directly). (Also, I do have a new `./go` script template that avoids the need to do this; can discuss it with you if you want.)